ui/apos/components/AposExportModal.vue

```diff
@@ -141,7 +141,11 @@ export default {
     },
     checked: {
       type: Array,
-      default: () => []
+      default: null
```

Comment:
why do you set it to `null`?
having an array by default avoids us having to make loads of conditions when checking the length

Reply:
Because checked is used only in batch operations, otherwise we user `this.doc`.
We need a way to now we are in batch operation or in a single piece operation.
I could make a check on `this.doc` instead maybe.

Reply:
I updated to use a data key `selectedDocIds` to store ids from `this.doc` or from `this.checked`